client/src/containers/mealDetail.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class MealDetail extends Component {
+  render() {
+    if (!this.props.meal) {
+      return <div>Select a meal to get Started!</div>;
+    }
+
+    return (
+      <div>
+        <h3>{this.props.meal.name}</h3>
```

Comment:
Our meal object has slightly different property names than what you have here. Make sure to look over the meal object found in `/src/models/meal.js` to see how the property names are spelled.